invoices/templates/personweek.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
```

Comment:
These seem to be almost exact copies of per-invoice report pages. If that is the case, please split common part to a separate file and include it.